db/migrate/20211202172358_add_repository_mirroring_policy.rb

```diff
@@ -0,0 +1,23 @@
+class AddRepositoryMirroringPolicy < ActiveRecord::Migration[6.0]
+  def up
+    add_column :katello_root_repositories, :mirroring_policy, :string
+
+    Katello::RootRepository.where(:content_type => 'yum').where(:mirror_on_sync => true).update_all(:mirroring_policy => ::Katello::RootRepository::MIRRORING_POLICY_COMPLETE)
+    Katello::RootRepository.where.not(:content_type => 'yum').where(:mirror_on_sync => true).update_all(:mirroring_policy => ::Katello::RootRepository::MIRRORING_POLICY_COMPLETE)
```

Comment:
Would `Katello::RootRepository.where(:mirror_on_sync => true).update_all(:mirroring_policy => ::Katello::RootRepository::MIRRORING_POLICY_COMPLETE)` get the same result as these two lines?

Reply:
ah yes, i meant to make custom repos content_only!!